sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/FunctionRegistry.scala

```diff
@@ -511,6 +511,11 @@ object FunctionRegistry {
     expression[BitwiseOr]("|"),
     expression[BitwiseXor]("^"),
 
+    expression[BitwiseAnd]("bitand"),
+    expression[BitwiseNot]("bitnot"),
+    expression[BitwiseOr]("bitor"),
+    expression[BitwiseXor]("bitxor"),
```

Comment:
Hi, @lipzhu . Thank you for contribution. For the aliases, we put them together like the following.
```scala
expression[BitwiseAnd]("&"),
expression[BitwiseAnd]("bitand"),
expression[BitwiseNot]("~"),
expression[BitwiseNot]("bitnot"),
expression[BitwiseOr]("|"),
expression[BitwiseOr]("bitor"),
expression[BitwiseXor]("^"),
expression[BitwiseXor]("bitxor"),
```

Reply:
How about modifing the function a bit so that we could assign expression with multiple names?  
```
expression[BitwiseAnd]("&", "bitand", ...),
expression[BitwiseNot]("~", "bitnot", ...),
...
```